tests: multiply acceptable failures by number of workers for MR DRT

otan opened the pull request:
TPC-C has 10 workers, and we're allowing each worker to die once during
a chaos test as the query gets cancelled by DistSQL. As such, tweak the
acceptable failure rate.

Resolves #72235 
Resolves #72234

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/72273)
<!-- Reviewable:end -->

otan committed:
tests: multiply acceptable failures by number of workers for MR DRT

TPC-C has 10 workers, and we're allowing each worker to die once during
a chaos test as the query gets cancelled by DistSQL. As such, tweak the
acceptable failure rate.

Release note: None

otan commented:
bors r=rafiss

otan commented:
bors r=rafiss
